app/src/main/java/org/mozilla/focus/fragment/CrashReporterFragment.kt

```diff
@@ -10,19 +10,23 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.appcompat.content.res.AppCompatResources
 import androidx.fragment.app.Fragment
-import kotlinx.android.synthetic.main.fragment_crash_reporter.*
 import mozilla.components.service.glean.private.NoExtras
 import org.mozilla.focus.GleanMetrics.CrashReporter
 import org.mozilla.focus.R
+import org.mozilla.focus.databinding.FragmentCrashReporterBinding
 
 class CrashReporterFragment : Fragment() {
     var onCloseTabPressed: ((sendCrashReport: Boolean) -> Unit)? = null
+    private lateinit var binding: FragmentCrashReporterBinding
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? = inflater.inflate(R.layout.fragment_crash_reporter, container, false)
+    ): View {
+        binding = FragmentCrashReporterBinding.inflate(inflater, container, false)
```

Comment:
The binding instance needs to be freed in `onDestroyView`.
But better yet, if we only need access to views inside `onViewCreated` then we don't need to keep around the binding instance, and the ceremony for freeing it.

We can have
```
class CrashReporterFragment : Fragment(R.layout.fragment_crash_reporter) {
    // no need for the `onCreateView` override since the ctor receiving a layout will itself call `onCreateView`
    ...
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val binding = FragmentCrashReporterBinding.bind(view)
        ...
    }
}
```